sys/src/go.mod

```diff
@@ -0,0 +1,3 @@
+module harvey-os.org
```

Comment:
Why do we need this file? The module at this url points to the [go repo](https://github.com/Harvey-OS/go/blob/98bcd60fd161e48f21dae558a19bec6360039962/go.mod#L1):
```
$ curl -L 'http://harvey-os.org/?go-get=1'
<html><head><meta name="go-import" content="harvey-os.org git https://github.com/Harvey-OS/go"></head></html>
```

Reply:
accidental commit :/ thanks for the heads up :)